engine/src/main/resources/templates/Common/insightsEmailBody.html

```diff
@@ -0,0 +1,520 @@
+                                                </table>
+                                            </td>
+                                        </tr>
+                                    {#else}
+                                        <tr>
+                                            <td class="rh-content__block" style="padding: 0px 0px 24px 0px;"/>
+                                        </tr>
+                                    {/if}
+                                </table>
+                            </td>
+                        </tr>
+                    {/if}
+
+                    {#if renderSection2}
+                    <tr>
+                        <td class="rh-body">
+                            <table class="rh-content" role="presentation" align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+                                <tr>
+                                    <td>
+                                        <table border="0" cellpadding="0" cellspacing="0" width="100%">
+                                            <tr>
+                                                <td class="rh-masthead__section__head">
+                                                    <p>{#insert content-title-section2}{/}<br/>
+                                                        <span class="rh-masthead__section__head__subtitle">{#insert content-subtitle-section2}{/}</span>
+                                                    </p>
+                                                </td>
+                                                <td class="rh-masthead__section__head">
+                                                    <p><span class="rh-masthead__section__head__counter">{#insert content-title-right-part-section2}{/}</span>
+                                                </td>
+                                            </tr>
+                                        </table>
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td class="rh-content__block" style="padding: 0px 0px 24px 0px;"/>
+                                </tr>
+                                <tr>
+                                    <td class="rh-content__block">
+                                        {#insert content-body-section2}{/}
+                                    </td>
+                                </tr>
+                                {#if renderButtonSection1}
+                                <tr>
+                                    <td class="rh-content__block" style="padding: 24px 0px;">
+                                        <table align="center">
+                                            <tr>
+                                                <td class="rh-cta-link" align="center">
+                                                    {#insert content-button-section1}{/}
```

Comment:
Shouldn't this be `content-button-section2`?

Reply:
Sure